### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by nrmay**
Publish SC Meeting #7 and #9. 
Add link for SORSE.

### CODE REVIEW

File: _posts/2020-08-01-SORSE-Call-For-Contributions.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+You can make contributions, about technical or soft skills, in any of the following formats:
+
+- [Panels and Discussions](https://sorse.github.io/programme/panels/)
+- [Posters and Lightning talks](https://sorse.github.io/programme/posters/)
+- [Software demos](https://sorse.github.io/programme/software-demos/)
+- [Talks](https://sorse.github.io/programme/talks/)
+- [Workshops](https://sorse.github.io/programme/workshops/)
+- [Blog Posts](https://sorse.github.io/faq/howto/contribute-blog-posts)
+
+This is an open continuous call with rolling deadlines at the end of the last day (UTC) of each month, and the next deadline is set for 1st September.
```
Comment (manodeep) [inline _posts/2020-08-01-SORSE-Call-For-Contributions.markdown:21]: Probably should be 1st September

File: _posts/2020-08-04-Minutes-of-SCM-08.markdown [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- [NM]	Schedule a special meeting to discuss eResearch Australasia 2020, and create a discussion document to share ideas.
+- [NM]	Share 2019 steering committee election process and dates.
+- [All] 	Expand ideas submitted to the poll: https://www.tricider.com/brainstorming/33t2mlX7PGN
+- [NL] 	Planning our first online event about:
+          	"How to publicly share 100 TB's of data with a Jupyter Notebook access"
+- [All] Create posts for the website about outputs/results from the following:
+  - [NM] Create google docs for collaboration on posts.
+  - [HP] Oversee post: Workshop and BoF at eResAus 2019 
+  - [NL] Oversee post: eResNZ 2020.
+  - [MS] Share lists of attendees for eResAus 2019 events.
```
Comment (manodeep) [inline _posts/2020-08-04-Minutes-of-SCM-08.markdown:74]: The list of attendees needs to be assigned to someone else ([RL/NL]?)

Comment (manodeep) [reply _posts/2020-08-04-Minutes-of-SCM-08.markdown:74]: Ahh I see in the next minutes

File: _posts/2020-12-31-template.markdown [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (manodeep) [inline _posts/2020-12-31-template.markdown:1]: Is this file supposed to contain anything?

File: links.md [hunk 4]
```diff
@@ -8,54 +8,25 @@ permalink: /links/
 
 
 ### RSE Associations
```
Comment (manodeep) [inline links.md:10]: Would be good to add a link to the [RSE International](https://researchsoftware.org/)

**Comment by manodeep** [comment]
@nrmay I took a quick look and made a couple of suggestions. Looks good to me. 

@nooriyahp Will you please take a look and confirm?

File: _posts/2020-12-31-template.markdown (hunk 3 above)
Comment (nrmay) [reply _posts/2020-12-31-template.markdown:1]: No extra content. This is the default template that people can copy to create posts.

File: _posts/2020-08-01-SORSE-Call-For-Contributions.markdown (hunk 1 above)
Comment (manodeep) [reply _posts/2020-08-01-SORSE-Call-For-Contributions.markdown:21]: The deadline date keeps becoming stale. Perhaps, we should just take the actual date out, and say 1st of the month

File: links.md [hunk 5]
```diff
@@ -8,54 +8,25 @@ permalink: /links/
...
-
-  
+  - <a class="" target="_blank" href="https://society-rse.org/">Society of Research Software Engineering</a>
+  - <a class="" target="_blank" href="https://de-rse.org/">Germany RSE Association</a>
+  - <a class="" target="_blank" href="https://nl-rse.org/">Netherlands RSE Association</a>
+  - <a class="" target="_blank" href="https://nordic-rse.org/">Nordic RSE Association</a>
+  - <a class="" target="_blank" href="https://us-rse.org/">USA RSE Association</a>
+
+
 ### RSE Meetings
```
Comment (manodeep) [inline links.md:18]: May be we should rename to "Past RSE Meetings"

File: links.md [hunk 6]
```diff
@@ -8,54 +8,25 @@ permalink: /links/
...
-  - <a class=""  target="_blank" 
-  			href="https://society-rse.org/community/rse-leaders-meetings/">
-  			RSE Leaders Meetings
-  			</a>
+  - <a class=""  target="_blank" href="https://www.rseconference.nz/">New Zealand RSE Conference 2020</a>
+  - <a class=""  target="_blank" href="https://rsecon2020.society-rse.org/">RSE Conference 2020</a>
+  - <a class=""  target="_blank" href="https://rse.ac.uk/conf2019/">RSE Conference 2019</a>
+  - <a class=""  target="_blank" href="https://society-rse.org/community/rse-leaders-meetings/">RSE Leaders Meetings</a>
 
 ### Online Communities
```
Comment (manodeep) [inline links.md:24]: How about "Getting Involved with Online Communities"?

File: _posts/2020-08-01-SORSE-Call-For-Contributions.markdown (hunk 1 above)
Comment (nrmay) [reply _posts/2020-08-01-SORSE-Call-For-Contributions.markdown]: Removed specific deadline.

File: links.md (hunk 4 above)
Comment (nrmay) [reply links.md:10]: Added.

File: links.md (hunk 5 above)
Comment (nrmay) [reply links.md]: Done.

File: links.md (hunk 6 above)
Comment (nrmay) [reply links.md]: Done

**Pull request merged**